bedrock/basic/README.md

```diff
@@ -0,0 +1,10 @@
+# Basic Amazon Bedrock workflow
+
+A basic Bedrock workflow. Starts a workflow with a prompt, generates a response and ends the workflow.
+
+To run, first see [README.md](../README.md) for prerequisites. Then, run the following from this directory:
```

Comment:
```suggestion
To run, first see [README.md](../../README.md) for prerequisites. Then, run the following from this directory:
```

Do you want to point them to the base README for prereqs?

Reply:
Note @steveandroulakis this applies to all three sub-READMEs and not just this one